vms/avm/vm.go

```diff
@@ -252,7 +252,7 @@ func (vm *VM) GetTx(txID ids.ID) (snowstorm.Tx, error) {
  */
 
 // IssueTx attempts to send a transaction to consensus
-func (vm *VM) IssueTx(b []byte) (ids.ID, error) {
+func (vm *VM) IssueTx(b []byte, onDecide func(choices.Status)) (ids.ID, error) {
```

Comment:
Can you put a comment specifying what `onDecide` is?

Reply:
Updated